R/twitter_timeline.R

```diff
@@ -0,0 +1,20 @@
+#' Get a user's timeline
+#' @export
+getTwitterTimeline <- function(user, n=3200, includeRts = FALSE,
```

Comment:
What is the reason for 3200 for default n? Please leave the reason as a comment.